charts/prometheus-kafka-exporter/values.yaml

```diff
@@ -145,17 +145,20 @@ kafka:
 
 # Set security context for the kafka-exporter pod. Useful when you need to adapt to an existing PSP
 securityContext: {}
-  # runAsUser:
-  # runAsGroup:
-  # fsGroup:
+  # fsGroup: 2000
+  # runAsGroup: 10002
+  # runAsUser: 10001
+  # seccompProfile:
+  #   type: RuntimeDefault
 
 # Set securityContext for the kafka-exporter container
 containerSecurityContext: {}
-  # runAsUser: 10001
-  # runAsNonRoot: true
   # allowPrivilegeEscalation: false
   # capabilities:
   #   drop: ["all"]
   # readOnlyRootFilesystem: true
+  # runAsGroup: 10002
+  # runAsNonRoot: true
+  # runAsUser: 10001
   # seccompProfile:
```

Comment:
"seccompProfile" should not work in containercontext

Reply:
It works in both contexts as per API spec [securitycontext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#securitycontext-v1-core).